include/rocksdb/perf_context.h

```diff
@@ -26,6 +28,10 @@ struct PerfContextByLevel {
   // exist.
   uint64_t bloom_filter_full_true_positive = 0;
 
+  uint64_t user_key_return_count;  // total number of user key returned
```

Comment:
Need to have better comment. For example, explaining how it counts in deletion and merge cases.